index.html

```diff
@@ -0,0 +1,499 @@
+              <img
+                src="./img/yessica-christy.svg"
+                width="50"
+                height="50"
+                alt="yessica-christy"
+              />
+              <div class="trusted-customer__username">
+                <p class="trusted-customer__name">Yessica Christy</p>
+                <p class="trusted-customer__born">Shanxi, China</p>
+              </div>
+              <div class="trusted-customer__rating">
+                <p class="trusted-customer__text">4.5</p>
+                <img src="./img/star.svg" width="16" height="16" alt="star" />
+              </div>
+            </div>
+            <blockquote class="trusted-customer__text">
+              “I like it because I like to travel far and still can connect with
+              high speed.”.
+            </blockquote>
+          </div>
+          <div class="trusted-customer__item">
+            <div class="trusted-customer--datauser">
+              <img
+                src="./img/kim-young-jou.svg"
+                width="50"
+                height="50"
+                alt="kim-young-jou"
+              />
+              <div class="trusted-customer__username">
+                <p class="trusted-customer__name">Kim Young Jou</p>
+                <p class="trusted-customer__born">Seoul, South Korea</p>
+              </div>
+              <div class="trusted-customer__rating">
+                <p class="trusted-customer__text">4.5</p>
+                <img src="./img/star.svg" width="16" height="16" alt="star" />
+              </div>
+            </div>
+            <blockquote class="trusted-customer__text">
+              “This is very unusual for my business that currently requires a
+              virtual private network that has high security.”.
+            </blockquote>
+          </div>
+        </div>
+        <div class="trusted-customer__control">
+          <button class="button button__left" type="button">left</button>
+          <button class="button button__right" type="button">right</button>
+        </div>
+      </section>
+
+      <section class="subscribe">
+        <div class="subscribe__text">
+          <h4 class="subscribe__title">
+            Subscribe Now for Get Special Features!
+          </h4>
+          <p class="subscribe__subtitle">
+            Let's subscribe with us and find the fun.
+          </p>
+        </div>
+        <button class="button button__subscribe" type="button">
+          Subscribe Now
+        </button>
+      </section>
+    </main>
+    <footer class="footer">
+      <div class="footer__about">
+        <img
+          src="./img/logo.svg"
+          width="149"
+          height="36"
+          alt="lasslessvpn-logo"
+        />
+        <p class="footer__subtitle">
+          LaslesVPN is a private virtual network that has unique features and
+          has high security.
+        </p>
+        <div class="footer__socials">
+          <a class="footer__socials--link" href="#"
```

Comment:
Same here. This is a modifying class. You seem to need `footer__link` here instead.

Reply:
+